README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository contains testeth, an Ethereum consensus test generator.
 
-**`ethereum/testeth` is a temporal project until a new RPC based test generation tool is ready.**
+**This repository is out of date and should be closed. Anyone wanting to use `testeth` should instead build and run it from the ethereum/aleth repository.**
```

Comment:
which `this` ? is there any context in the document?

Reply:
I can add an explicit reference to `ethereum\testeth` but I wonder why you want it. In what situation will this file be read and the reader not know where it came from?

Reply:
oh sorry I thought it is the other repo. ok then. 
having that caption in red would be a nice.